bedrock/firefox/templates/firefox/share.html

```diff
@@ -0,0 +1,88 @@
+  {% set email_alt_text = "Per E-Mail gesendet" %}
+  {% set email_subject = "Treten Sie dem Team Firefox bei" %}
+  {% set sign_off = "<strong>Powered by Mozilla.</strong> <br> Für dich und das Web. Schon seit 1998."|safe %}
+
+{% elif LANG == "fr" %}
+
+  {% set main_title = "Invitez vos amis à rejoindre la <br> <strong>team Firefox</strong>"|safe %}
+  {% set page_description = "Nous pensons que, comme un bon ami, il est temps de les inviter à découvrir le bon côté de l'internet. Nous avons donc préparé quelque chose pour vous." %}
+  {% set main_tagline = "Nous avons préparé un message personnel pour vos amis, afin que vous n'ayez pas à le faire vous-même. Ils ne se douteront de rien, c'est promis !" %}
+  {% set share_message = "Salut ! Je voulais juste te dire que tu as de beaux yeux. Leur forme parfaite me rappelle le logo de Firefox. Tu connais Firefox ? Ce navigateur privé, super fiable. Il est plutôt pas mal en vrai, voici un lien si tu veux le tester : <a href='https://mzl.la/ctd-fr'>mzl.la/ctd-fr</a>.<br><br> Sinon, comment vas-tu ?"|safe%}
+  {% set share_message_email = "Salut ! Je voulais juste te dire que tu as de beaux yeux. Leur forme parfaite me rappelle le logo de Firefox. Tu connais Firefox ? Ce navigateur privé, super fiable. Il est plutôt pas mal en vrai, voici un lien si tu veux le tester : https://mzl.la/ctd-fr.%0D%0D Sinon, comment vas-tu ?"|safe%}
+  {% set share_cta = "Copier et partager" %}
+  {% set text_copied = "Texte copié !" %}
+  {% set other_socials = "Ou partager sur un des canaux suivants : " %}
+  {% set whatsapp_alt_text = "Partager avec WhatsApp" %}
+  {% set email_alt_text = "Envoyé par email" %}
+  {% set email_subject = "Rejoignez l'équipe Firefox" %}
+  {% set sign_off = "<strong>Conçu par Mozilla.</strong> <br> Pensé pour vous depuis 1998."|safe %}
+
+{% endif %}
+
+{% block content %}
+  <section class="mzp-l-content mzp-t-content-md firefox-share">
+    <div class="mzp-c-logo mzp-t-logo-lg mzp-t-product-firefox"></div>
+    <h1>{{ main_title }}</h1>
+    <p class="tagline">{{ main_tagline }}</p>
+
+    <div class="copy-wrapper mzp-t-content-sm">
+      <div class="copy-text">
+        <p id="text-to-copy">{{ share_message }}</p>
+      </div>
+      <button id="copy-button">
+        <span id="share-cta">{{ share_cta }}</span>
+        <span id="text-copied">{{ text_copied }}</span>
+      </button>
+    </div>
+
+    <div class="other-socials">
+      <p>{{ other_socials }}</p>
+      <div class="other-socials-wrapper">
+
+        {% if LANG == "de" %}
+          <a href="https://mzl.la/4bZN9ak" id="share-whatsapp"><img src="{{ static('img/firefox/share-page/whatsapp.svg') }}" alt=""><span>{{ whatsapp_alt_text }}</span></a>
+        {% elif LANG == "fr" %}
+          <a href="https://mzl.la/4a0zYE3" id="share-whatsapp"><img src="{{ static('img/firefox/share-page/whatsapp.svg') }}" alt=""><span>{{ whatsapp_alt_text }}</span></a>
+        {% endif %}
```

Comment:
**suggestion:**  super small suggestion, but could we avoid repeating the HTML here and use a variable for the `href` instead? Something like:

```jinja
{% if LANG == "de" %}
  {% set whatsapp_url = 'https://mzl.la/4bZN9ak' %}
{% elif LANG == "fr" %}
  {% set whatsapp_url = 'https://mzl.la/4a0zYE3' %}
{% endif %}

<a href="{{ whatsapp_url }}" id="share-whatsapp"><img src="{{ static('img/firefox/share-page/whatsapp.svg') }}" alt=""><span>{{ whatsapp_alt_text }}</span></a>
```

Reply:
omg i was getting errors when doing it this way before, but i must have missed something to make it go haywire. Thank you for fixing it!